phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2846,21 +2872,25 @@ private MetaDataMutationResult mutateColumn(
                 separateLocalAndRemoteMutations(region, tableMetadata, localMutations,
                         remoteMutations);
                 if (!remoteMutations.isEmpty()) {
-                    // there should only be remote mutations if we are adding a column to a view
+                    // there should only be remote mutations if we are updating the last ddl
```

Comment:
This comment is no longer valid right?

Reply:
Oops, forgot to undo that comment change. Thanks, will fix.